src/useGet.ts

```diff
@@ -61,32 +61,31 @@ export default function get(options: UseGetOptions): UseGetData {
     return isRef(params) ? params.value : params
   }
 
-  const servicePath = computed<string>(() => model.servicePath)
   const state = reactive<UseGetState>({
+    isPending: false,
+    hasBeenRequested: false,
+    hasLoaded: false,
+    error: null,
+    isLocal: local
+  })
+
+  const computes = {
     item: computed(() => {
       const getterId = isRef(id) ? id.value : id
       const getterParams = isRef(params)
         ? Object.assign({}, params.value)
         : params == null
         ? params
         : { ...params }
-      if (getterParams != null) {
-        return model.getFromStore(getterId, getterParams) || null
-      } else {
-        return model.getFromStore(getterId) || null
-      }
-      // return model.store.state[model.servicePath].keyedById[id.value] || null
-      // return model.getFromStore(id.value) || null
+      return model.getFromStore<M>(getterId, getterParams) || null
```

Comment:
I believe this will cause a breaking change.  This is definitely better-written code, but I had to write it the other way due to some quirk that I couldn't figure out... and now can't remember. ;). Would you mind restoring the more-verbose option?

Other than this part, I believe this is ready to merge.

Reply:
Addressed in bd96d6d